pkgs/tools/filesystems/gocryptfs/default.nix

```diff
@@ -2,7 +2,7 @@
 { stdenv, buildGoPackage, fetchFromGitHub, openssl, pandoc, pkgconfig }:
 
 let
-  version = "v1.6.1";
+  version = "v1.7";
```

Comment:
```suggestion
```

Reply:
Then use

```
pname = "gocryptfs";
version = "1.7";
```

below.

###### when `pname` and `version` are present they are concatenated to make `name`.

Reply:
Resolved with d5363cbd5ae94539f339da48a4e36dab64e0f181